### PR TITLE
media-plugin/vdr-[mn]* Pkgcheck fixes vol. 5

### PR DESCRIPTION
**Pull request by madmartin**
65ac8aefcfa5 G - Martin Dummer / media-plugins/vdr-noepg: update EAPI 7 -> 8
a57dccd93f32 G - Martin Dummer / media-plugins/vdr-noepg: drop 0.0.4
b12eaec84edf G - Martin Dummer / media-plugins/vdr-newsticker: fix pkgcheck issues
62f4d6e4bf89 G - Martin Dummer / media-plugins/vdr-mlist: update EAPI 7 -> 8
c21b58a654be G - Martin Dummer / media-plugins/vdr-menuorg: fix pkgcheck issues
8caec26cc023 G - Martin Dummer / media-plugins/vdr-menuorg: drop 0.5.1-r1
2ff03a0fe0c2 G - Martin Dummer / media-plugins/vdr-mplayer: update EAPI 7 -> 8

---

Please check all the boxes that apply:

- [X] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [X] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [X] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [X] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: media-plugins/vdr-newsticker/vdr-newsticker-0.0.4-r1.ebuild [hunk 1]
```diff
@@ -9,13 +9,12 @@ DESCRIPTION="VDR plugin: Show rdf Newsticker on TV"
...
-KEYWORDS="~amd64 ~x86"
-SLOT="0"
 LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
 
-BDEPEND="acct-user/vdr
-	media-video/vdr"
-DEPEND="${BDEPEND}"
+BDEPEND="acct-user/vdr"
```
Comment (ConiKost) [inline media-plugins/vdr-newsticker/vdr-newsticker-0.0.4-r1.ebuild:16]: Doesn't need this also be in ```RDEPEND```?

Comment (madmartin) [reply media-plugins/vdr-newsticker/vdr-newsticker-0.0.4-r1.ebuild:16]: Hello @ConiKost ,
basically correct, media-video/vdr includes the dependency for acct-user/vdr - so it has an indirect dependency (see COMMON_DEPEND in media-video/vdr .

As far as I understood, BDEPEND="acct-user/vdr" is necessary on the build host when there are commands like "chown vdr:vdr" in the ebuild?!?

File: media-plugins/vdr-mlist/vdr-mlist-1.0.2.ebuild [hunk 2]
```diff
@@ -1,20 +1,18 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
```
Comment (juippis) [inline media-plugins/vdr-mlist/vdr-mlist-1.0.2.ebuild:4]: This requires a revbump, especially with stable keywords.

**Review by juippis — APPROVED** [review]
Remember to revbump in future. I saw you were doing that in the previous PR though.

**Pull request closed without merging**